packages/circuit-ui/components/Dialog/Dialog.spec.tsx

```diff
@@ -166,42 +166,79 @@ describe('Dialog', () => {
       render(<Dialog {...props} open />);
       vi.runAllTimers();
       expect(screen.getByText('Dialog content')).toBeVisible();
+      expect(screen.getByRole('button', { name: 'Close' })).toBeInTheDocument();
```

Comment:
`.toBeVisible()` is a bit stricter/safer since it ensures that the button isn't visually hidden.

```suggestion
      expect(screen.getByRole('button', { name: 'Close' })).toBeVisible();
```